### DISCUSSION TITLE
Time goes backwards

### DISCUSSION THREAD

**Discussion by georgep973** [EnigmaticaModpacks/Enigmatica2Expert#2167] [Q&A] [answered]
Every 4 seconds, the clock will go back 3 seconds. For example, my time will be at 09:45, go up to 09:48, and then instead of going to 09:49 it goes to 09:46. Every time it does this, the game freezes for a split second and continues, and it causes the world to seem as though its lagging even though technically I'm not dropping frames. I've tried some regular lag fixes such as adding command line arguments, lowering allocated ram from 10 gigs to 8, nothing works.

**Comment by HeatherComputer** [comment]
This should really be in the discord - that being said, this does sound like pretty hefty lag. What does `/spark healthreport` output?

**Reply by georgep973** [reply]
https://github.com/EnigmaticaModpacks/Enigmatica2Expert/assets/51968939/e2b4afbd-ae8a-40c9-997a-5543028c18bc
Didn't know there were some things that were meant for discord, sorry for that.

**Reply by HeatherComputer** [reply]
Welp, that answers that question. VERY laggy - time to get a profile.
`/spark profiler --timeout 50` will spend 50 seconds profiling, then dump a link in chat.

**Reply by georgep973** [reply]
https://spark.lucko.me/mX3bAX9ioc

**Reply by ReiDaTecnologia** [reply]
Classic Mekanism...

**Reply by HeatherComputer** [reply]
Over 90% of your tick time is spent on mekanism's logistical transporters simulating inserts into ic2 recyclers.

Both of these are pretty bad - though the recyclers here are the main concern, with them being most of the tick time within the transporter update method.
How many of these things do you have?

**Reply by georgep973** [reply]
That is my scapper maker for the ic2 recyclers. I have 8 recyclers each with 12 overclock upgrades and they are being pumped with cobblestone from the nuclearcraft generators. Everything with that setup is connected with mekanism cables, ultimate universal and ultimate logistcal cables. About half a stack for each set of cables is being used for that.

**Reply by HeatherComputer** [reply, marked as answer]
Interesting. Logistical transporters aren't all too efficient, I'd suggest you try swapping them for conduits - or alternatively remove the item pipes altogerher, as the cobblestone generators auto eject.

In the long run, I'd advise keeping scrap production and usage to a minimum - if this is for the matter fabricator, you can feed it with 4mrf/t and that maxes its speed with no scrap usage. (at high power levels, the matter fabricator in general just sees diminishing returns with scrap)

**Reply by georgep973** [reply]
Getting rid of the logistical solved the issue, I am currently working on upgrading the power so that I don't need the scraps (environmental tech solar panels go brr). Thank you for the help again, if I have issues like that in the future I guess go to the discord correct?

**Reply by HeatherComputer** [reply]
Yep, correct. There's a support channel there that's much better suited to less permanent discussions like this one, and more people there that are willing to help out.
